src/Compilers/Core/Portable/MemberDescriptor.cs

```diff
@@ -72,7 +72,7 @@ public string DeclaringTypeMetadataName
 
         public MemberDescriptor(
             MemberFlags Flags,
-            byte DeclaringTypeId,
+            short DeclaringTypeId,
             string Name,
             ImmutableArray<byte> Signature,
             ushort Arity = 0)
```

Comment:
@AlekseyTs Yes, I'll hold off on renaming to lowercase. Good to go other than that?